cpp/ps0004/Task4.cpp

```diff
@@ -41,3 +44,4 @@ int main()
 
     return 0;
 }
+// great work 😎 9.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors, bugs or warnings while compilation and execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case
you can check the [problem solution](https://onlinegdb.com/9x6wLTlkK) for more